test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -4093,9 +4094,41 @@ def test_vnodepernuma_use_hyperthreads(self):
         b1 = {'job_state': 'Q'}
         self.server.expect(JOB, b1, jid2)
 
+    def test_cgroup_default_config(self):
+        """
+        Test to make sure using the default hook config file
+        still run a basic job, and cleans up cpuset upon qdel.
+        """
+        # The default hook config has 'memory' subsystem enabled
+        if not self.paths['memory']:
+            self.skipTest('Test requires memory subystem mounted')
+        self.load_default_config()
+        # Reduce the noise in mom_logs for existence=False matching
+        c = {'$logevent': '511'}
+        self.mom.add_config(c)
+        a = {'Resource_List.select': 'ncpus=1:mem=100mb'}
+        j = Job(TEST_USER, attrs=a)
+        j.create_script(self.sleep15_job)
+        jid = self.server.submit(j)
+        a = {'job_state': 'R'}
+        self.server.expect(JOB, a, jid)
```

Comment:
You can combine the above two lines if you want.  They will fit <79 chars

Reply:
Good point.